lib/db.js

```diff
@@ -1188,7 +1175,7 @@ module.exports = (
   // as fixed indices into arrays instead. Takes an unpacked session tokens
   // structure as its argument, returns the packed structure.
   function packTokensForRedis (tokens) {
-    return Object.keys(tokens).reduce((result, tokenId) => {
+    return JSON.stringify(Object.keys(tokens).reduce((result, tokenId) => {
```

Comment:
I should have returned the stringified result when I first wrote this to make it similar to the unpack function, which calls `JSON.parse` on entry. The fact that it's now called in three different places just made that more pressing.